app/models/set_member_subject.rb

```diff
@@ -4,7 +4,6 @@ class SetMemberSubject < ActiveRecord::Base
   
   belongs_to :subject_set, counter_cache: true, touch: true
   belongs_to :subject
-  has_many :classifications
```

Comment:
Do you think we'd have a storm topology ready and load tested for stargazing (March 20th so really late feb early march)? It adds a little overhead to the infrastructure. 

I assume the storm spout would be the kafka consumer for 'classification' topic? How do you handle storm failures and starting from the correct topic offset to ensure we get correct counts?

I think the most basic working version of this is a background worker (classification lifecycle) that manually inc's all the classification counts for the subject SMS's for that workflow.